didact/src/components/courses/Lessons.jsx

```diff
@@ -1,9 +1,7 @@
 import React, {useState, useEffect} from 'react'
 import Lesson from './Lesson'
 
-const Lessons = ({section, props, lessons}) => {
-    console.log('props in lesson array: ', props)
-
+const Lessons = ({course, section, props, lessons}) => {
```

Comment:
Same as above. Maybe bring in props (since you're passing it to lesson), and destructure inside?
const {course, section, lessons} = props